src/content/app/entity-viewer/state/api/queries/geneOverviewQuery.ts

```diff
@@ -46,6 +49,7 @@ export type GeneOverview = Pick<
       NonNullable<FullGene['metadata']['name']>,
       'accession_id' | 'url'
     > | null;
+    biotype: Pick<NonNullable<FullGene['metadata']['biotype']>, 'value'> | null;
```

Comment:
is `biotype` non nullable?  if so why we add `| null` ?

Reply:
That's fixed now ✅